webpack.config.js

```diff
@@ -43,21 +44,11 @@ otherScripts.forEach( function ( script ) {
 const webpackConfig = getBaseWebpackConfig(
 	{ WP: true },
 	{
-		entry: wizardsScriptFiles,
+		entry: {
+			...wizardsScriptFiles,
+			blocks: path.join( __dirname, 'assets', 'blocks', 'index.js' ),
+		},
 	}
 );
 
-// overwrite Calypso's optimisation
-webpackConfig.optimization = {
-	splitChunks: {
-		cacheGroups: {
-			commons: {
-				name: 'commons',
-				chunks: 'initial',
-				minChunks: 2,
-			},
-		},
-	},
-};
-
```

Comment:
As I was scaffolding this draft I struggled a bit to get the `registerBlockType` function, from `@wordpress/blocks`, to work without error. Turns out I had to remove these optimizations to get rid of the following error:

```
"Cannot read properties of undefined (reading '__esModule')"
```

I'd appreciate some insight. cc @adekbadek

Reply:
Assets are [chunked](https://webpack.js.org/guides/code-splitting/) to avoid repetition. `Newspack::load_common_assets();` was missing. Fixed in [82d0895](https://github.com/Automattic/newspack-plugin/pull/1643/commits/82d089590743fa69fea953b1b7d943f6fe664504)

Reply:
Thank you!